### PR TITLE
KI: Cube freezes on 'Connected!' loading screen

### PR DESCRIPTION
**Pull request by EdithLansens**
@AlexanderGorbunovSkyline, is this OK to merge?

### CODE REVIEW

File: user-guide/Troubleshooting/Known_issues/KI_Cube_connection_issue_alarm_tabs.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# Cube freezes on 'Connected!' loading screen when no alarm tabs are displayed
+
+## Affected versions
+
+DataMiner versions 10.3.9 and 10.3.10.
+
+## Cause
+
+By default, the [three default alarm tabs](xref:ChangingTheAlarmConsoleLayout#the-three-default-alarm-tabs) are displayed at startup: active alarms, information events, and masked alarms. However, when the Alarm Console settings have been configured to display no tabs in the Alarm Console, a startup issue may occur.
```
Comment (AlexanderGorbunovSkyline) [inline user-guide/Troubleshooting/Known_issues/KI_Cube_connection_issue_alarm_tabs.md:13]: "Startup issue" usually implies a server startup issue. In this case, "a connection issue" is more accurate.

File: user-guide/Troubleshooting/Known_issues/KI_Cube_connection_issue_alarm_tabs.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Right-click the *Tab pages* column, and select *Add tab page > Active alarms*.
+
+1. Save your changes by clicking *Apply* in the lower right corner.
+
+1. Reconnect to Cube.
+
+## Fix
+
+Install DataMiner 10.3.11<!--RN 37436-->.
```
Comment (AlexanderGorbunovSkyline) [inline user-guide/Troubleshooting/Known_issues/KI_Cube_connection_issue_alarm_tabs.md:31]: yes, but DataMiner 10.3.11 will be released only this Friday (October 20).

File: user-guide/Troubleshooting/Known_issues/KI_Cube_connection_issue_alarm_tabs.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+uid: KI_Cube_connection_issue_alarm_tabs
+---
+
+# Cube freezes on 'Connected!' loading screen when no alarm tabs are displayed
+
+## Affected versions
+
+DataMiner versions 10.3.9 and 10.3.10.
```
Comment (EdithLansens) [inline user-guide/Troubleshooting/Known_issues/KI_Cube_connection_issue_alarm_tabs.md:9]:
```suggestion
- DataMiner versions 10.3.9 and 10.3.10.

- Any versions from DataMiner 10.2.0 onwards using automatic client updates
```

Is this better, @AlexanderGorbunovSkyline?

File: user-guide/Troubleshooting/Known_issues/KI_Cube_connection_issue_alarm_tabs.md (hunk 2 above)
Comment (EdithLansens) [reply user-guide/Troubleshooting/Known_issues/KI_Cube_connection_issue_alarm_tabs.md]: I have added this to the list of current issues for now and will update it with the fix and move it to resolved issues this Friday!

File: user-guide/Troubleshooting/Known_issues/KI_Cube_connection_issue_alarm_tabs.md (hunk 3 above)
Comment (AlexanderGorbunovSkyline) [reply user-guide/Troubleshooting/Known_issues/KI_Cube_connection_issue_alarm_tabs.md:9]: Looks good to me, thanks!

**Pull request merged**